README.md

```diff
@@ -121,11 +121,9 @@ the code also follows GlobalPlatform standards. The exceptions are as follows:
 	against those projects later on and we don't expect that it is easy to
 	convince other software projects to change coding style.
 
-Regarding the checkpatch tool, since it is licensed under the terms of GNU GPL
-License Version 2, we
-cannot include this script directly into this project. Please use checkpatch
-directly from the Linux kernel git in combination with the local [checkpatch
-script].
+Regarding the checkpatch tool, it is not included directly into this project.
+Please use checkpatch.pl from the Linux kernel git in combination with the
+local [checkpatch script].
```

Comment:
At some point, do we plan on including it? Maybe the first time it makes sense to modify it?

Reply:
> Maybe the first time it makes sense to modify it?

Exactly. Currently it is easy enough to reference an external copy, and we don't need to make any  modification, so we'd better avoid the burden of maintaining it.